src/extensions/additional/QuoteLink/QuoteLinkSpecs/index.ts

```diff
@@ -0,0 +1,75 @@
+import {transform as quoteLink} from '@diplodoc/quote-link-extension';
+import type {Node} from 'prosemirror-model';
+
+import type {ExtensionAuto} from '#core';
+import {nodeTypeFactory} from 'src/utils';
+
+import {moveLinkToQuoteAttributes} from './md/moveLinkToQuoteAttributes';
+
+export const quoteLinkNodeName = 'yfm_quote-link';
+export const quoteLinkType = nodeTypeFactory(quoteLinkNodeName);
+export const isQuoteLinkNode = (node: Node) => node.type.name === quoteLinkNodeName;
+
+export enum QuoteLinkAttr {
+    Href = 'href',
```

Comment:
the `cite` attribute must be used insted of `href`
https://www.w3schools.com/tags/att_blockquote_cite.asp